### ISSUE TITLE
Address existing accessibility issues

### ISSUE THREAD

**Issue by sbearcsiro** [AtlasOfLivingAustralia/volunteer-portal#514]
Whilst evaluating accessibility of the audio transcription template, we discovered some pre-existing A11Y issues with the site that should be addressed.  The A11Y report is here: https://googlechrome.github.io/lighthouse/viewer/?gist=030ca448804b51b920f02ce8f79243e4

The following sections of the linked report contain the pre-existing issues:

- "Image elements do not have [alt] attributes"
- "Form elements do not have associated labels"
- "[aria-hidden="true"] elements contain focusable descendents"
- "Background and foreground colors do not have a sufficient contrast ratio."
- "[id] attributes on active, focusable elements are not unique"

**Comment by JennaMalkin** [comment]
This issue should be split into different areas to be implemented usefully.
